app/views/unattended/provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -0,0 +1,31 @@
+<%#
+kind: iPXE
+model: ProvisioningTemplate
+name: iPXE intermediate script
+snippet: false
+-%>
+#!ipxe
+
+# Skips booting from network and continues booting from next device
+exit
```

Comment:
Is this intentional? This always exits, not sure what this means.

Reply:
This looks like a leftover. The actual template begins below.

Reply:
Yep, sorry will fix